### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard:Focus is not visible on the color radio buttons.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9178]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.4.7;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.7

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
4. Navigate to the color radio buttons using tab key.

**Actual:**
On navigating to the radio button using the tab key the focus is not visible as well as using arrow key for navigation.

**Expected:**
The focus should be clearly visible in the form of dotted lines or a dark outline around the buttons.

**User Impact:**
Keyboard users will not know which button they are currently on.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EW-CgB3b7dlMmo7a7al6-vcBpIXLXMuZC8Zsz1nbkPggHQ?e=VVSbom

**Comment by alanrenmsft** [comment]
dupe of #9223

**Comment by Roopesh-T** [comment]
The above issue is not repro in the env
![image](https://user-images.githubusercontent.com/86601127/217237240-2c696743-c71e-4709-ab73-4279bd61778d.png)
so closing the bug.
![9178_Closed](https://user-images.githubusercontent.com/86601127/217237269-7e33fb66-1095-449c-9b66-4cba601d3004.gif)
GitHub Tag:#closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
